pkg/tsdb/testdata/scenarios_test.go

```diff
@@ -0,0 +1,37 @@
+package testdata
+
+import (
+	"testing"
+	"time"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/tsdb"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestTestdataScenarios(t *testing.T) {
+	Convey("random walk ", t, func() {
+		if scenario, exist := ScenarioRegistry["random_walk"]; exist {
+
```

Comment:
Seems weird to test that this exists in a test?

Anyway, you can flip this around and call `https://golang.org/pkg/testing/#T.Fatal` to exit early if the scenario doesn't exist.

Reply:
👍